docs/guides/analyzing-clustered-experiments.md

```diff
@@ -42,8 +42,9 @@ illustrate this, we will outline two examples.
 _Average order value_ is one of the examples mentioned in the [Ratio Metrics](/data-management/metrics/ratio-metric)
 documentation. Although we can recognize that it is a Ratio
 Metric because it is normalized by a count of orders rather than a count of assigned users (which would make it a Simple
-Metric), it is also a good example of a clustered metric. Understanding how this familiar ratio metric is a clustered
-analysis provides the intuition for how Ratio Metrics accommodate more complicated clustered analyses.
+Metric), it is also a good example of a clustered metric. This makes this example a particularly helpful introduction
+because customers who use AOV are already running clustered experiment analyses, even if they do not realize it.
```

Comment:
Minor, but the third person here ("customers", "they") feels off. here feels off to me. It would make sense if the doc were directed at Eppo employees; but it's primarily for the customers themselves.

Reply:
Good call! Second person feels awkward here too, so I just avoided using a pronoun. https://github.com/Eppo-exp/eppo-docs/commit/6d471c9cb2c79c953f7d0778a0abfb4b3b1c27e3